src/vm/compile.cpp

```diff
@@ -7290,6 +7306,81 @@ HRESULT CompilationDomain::SetPlatformWinmdPaths(LPCWSTR pwzPlatformWinmdPaths)
 
     return S_OK;
 }
+
+class MethodsForStubEnumerator
+{
+    SHash < NoRemoveSHashTraits<MapSHashTraits<CORINFO_METHOD_HANDLE, CORINFO_METHOD_HANDLE>>>::KeyIterator current;
```

Comment:
The template spacing bug was fixed long ago.

Reply:
True. Copy/Paste is the best. Fixed